v1/frontend/package.json

```diff
@@ -1,91 +1,51 @@
 {
   "name": "imtdb",
-  "version": "1.0.0",
-  "description": "IMTDb frontend application",
-  "author": "Johan van Iperen <johan@johanvaniperen.nl>",
+  "version": "0.2.0",
```

Comment:
Waarom is de versie verlaagd?

Reply:
Komt door de Vue CLI, die zet hem standaard om 0.1.0. Ik heb hem opgehoogd naar 0.2.0